x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -225,7 +227,7 @@ const TGridStandaloneComponent: React.FC<TGridStandaloneProps> = ({
     docValueFields: [],
     entityType,
     excludeEcsData: true,
-    fields,
+    fields: additionalFieldsToFetch ? [...fields, ...additionalFieldsToFetch] : fields,
```

Comment:
timeline does have a property that already exists to do this, and for the same reason, supporting the actions column: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx#L350 so instead of building fields here, could you pass additionalFieldsToFetch as queryFields in this file below something like 
`    dispatch(
      tGridActions.initializeTGridSettings({
        id: STANDALONE_ID,
        defaultColumns: columns,
        queryFields: additionalFieldsToFetch,
        footerText,
        sort,
        loadingText,
        unit,
      })
    );` 
    That property is already used when creating fields above. Also using '*' might be sort of expensive, maybe just pass an array of the fields used in the flyout instead.

Reply:
I replaced `additionalFieldsToFetch` prop with `queryFields` and added it to the `tGridActions.initializeTGridSettings` with the other props being passed in.